File: rust/Python3/RustParserBase.py
from antlr4 import *
try:
    from .RustLexer import RustLexer
except ImportError:
    from RustLexer import RustLexer


class RustParserBase(Parser):

    def NextGT(self) -> bool:
        return self._input.LA(1) == RustLexer.literalNames.index(f"'>'")

    def NextLT(self) -> bool:
        return self._input.LA(1) == RustLexer.literalNames.index(f"'<'")

File: rust/Java/RustParserBase.java
import org.antlr.v4.runtime.*;

public abstract class RustParserBase extends Parser {
    public RustParserBase(TokenStream input){
        super(input);
    }

    public boolean NextGT() {
        return _input.LA(1) == RustParser.GT;
    }

    public boolean NextLT() {
        return _input.LA(1) == RustParser.LT;
    }
}